build.sbt

```
@@ -1,8 +1,8 @@
 import com.typesafe.sbt.packager.docker.Cmd
 
-val scala212 = "2.12.15"
+//val scala212 = "2.12.15"
 val scala213 = "2.13.6"
-val supportedScalaVersions = List(scala212, scala213)
+val supportedScalaVersions = List(scala213)
```

```suggestion
val supportedScalaVersions = List(scala212)
```

I think you're doing the opposite of what you're trying to do here :sweat_smile:

---

totally 🤦